### PR TITLE
HOS-65: Fix notices bug when < 2, fix create notice additional request

### PR DESCRIPTION
**Pull request by yana617**

### CODE REVIEW

File: src/components/notices-control-view/Notice.vue [hunk 1]
```diff
@@ -63,7 +63,9 @@ export default {
   },
   components: { Button },
   created() {
-    this.$store.dispatch('notices/getNoticeById', { _id: this.noticeId });
+    if (!this.title) {
```
Comment (Aphelmion) [inline src/components/notices-control-view/Notice.vue:66]: WHy?

File: src/store/modules/notices.js [hunk 2]
```diff
@@ -33,6 +33,7 @@ const actions = {
     }
   },
   createNotice: async ({ commit }, params = {}) => {
+    commit(SET_CREATE_ERRORS, []);
```
Comment (Aphelmion) [inline src/store/modules/notices.js:36]: what is SET_CREATE ERRORS?

File: src/views/Home.vue [hunk 3]
```diff
@@ -11,7 +11,7 @@
           :noticeId="noticeId"
           v-bind="notices.data[noticeId]"
         />
-        <button @click="showAllNotices = !showAllNotices" class="home__notices__load-all-btn">
+        <button v-if="showLoadAllBtn" @click="showAllNotices = !showAllNotices" class="home__notices__load-all-btn">
```
Comment (Aphelmion) [inline src/views/Home.vue:14]: WHy showAllNotices = !showAllNotices

File: src/views/Home.vue [hunk 4]
```diff
@@ -58,6 +58,9 @@ export default {
       }
       return this.notices.list;
     },
+    showLoadAllBtn() {
```
Comment (Aphelmion) [inline src/views/Home.vue:61]: Load all what things?

File: src/components/notices-control-view/Notice.vue (hunk 1 above)
Comment (yana617) [reply src/components/notices-control-view/Notice.vue:66]: If we have title -> we have all data for current notice -> we do not need to load this again

File: src/store/modules/notices.js (hunk 2 above)
Comment (yana617) [reply src/store/modules/notices.js:36]: Action type (redux-same flow), if interested read about store in js apps (redux, vuex)

File: src/views/Home.vue (hunk 3 above)
Comment (yana617) [reply src/views/Home.vue:14]: We have showAllNotices = false by default
If we click it makes true and shows all notices
If we want to collapse list - click again and make value false
-> showAllNotices = !showAllNotices

File: src/views/Home.vue (hunk 4 above)
Comment (yana617) [reply src/views/Home.vue:61]: notices, will fix

**Pull request merged**